source/reference/color_schemes.rst

```diff
@@ -0,0 +1,256 @@
+=============
+Color Schemes
+=============
+
+
+Overview
+========
+
+Color schemes define the colors
+used to highlight source code in Sublime Text views
+and the colors use for different elements
+found in the editing area:
+background, foreground, selection, caret, etc.
+
+
+File Format
+===========
+
+Color scheme files use the Property List format
+and have the .tmTheme extension.
+The file names of color scheme files
+are displayed in the **Preferences → Color Scheme** menu.
+
+The file format of color schemes
+is inherited from Textmate.
+
+.. note::
+
+   Sublime Text uses the .tmTheme extension for color scheme files
+   to maintain compatibility with Textmate.
+   Confusingly, Sublime Text also has a notion
+   of a user interface (UI) theme.
+   A UI theme is a set of styles and decorations
+   to alter the look of the editor’s UI.
+   It’s important to remember
```

Comment:
I don't think we have used `’` as the actual apostrophe before (or at least I didn't). We might want to decide for a standard regarding this.